lib/improver/tests/test_set_up_test_cubes.py

```diff
@@ -351,24 +351,28 @@ def test_defaults(self):
         self.assertEqual(thresh_coord.name(), 'air_temperature')
         self.assertEqual(thresh_coord.var_name, 'threshold')
         self.assertEqual(thresh_coord.units, 'K')
-        self.assertEqual(len(result.attributes), 1)
-        self.assertEqual(result.attributes['relative_to_threshold'], 'above')
+        self.assertEqual(len(thresh_coord.attributes), 1)
+        self.assertEqual(
+            result.coord(var_name="threshold"
+                         ).attributes['spp__relative_to_threshold'], 'above')
```

Comment:
Why write this one out long form rather than just use the ```thresh_coord```?

Reply:
Good point, I've updated this to be `thresh_coord`.